vignettes/chapter2_Pipeline.Rmd

```diff
@@ -30,14 +29,14 @@ GGIR, has a computational structure of six parts which are applied sequentially
 
 -   **Part 1:** Loads the data and stores derived signal features or also known as metrics as needed for the following parts.
 -   **Part 2:** Basic data quality assessment and description per day and per file.
--   **Part 3:** Estimation of sustained inactivity bouts (rest periods) and sleep, needed for input to Part 4.
--   **Part 4:** Labels the sustained inactive periods derived in Part 3 as sleep, or daytime sustained inactivity, per night and per file.
--   **Part 5:** Compiles time series with classification of sleep and physical activity intensity categories by re-using information derived in part 2, 3 and 4. Next, Part 5 generates a describe summary such as total time in intensity categories, the number of bouts, time spent in bouts and average acceleration per category, and fragmentation.
--   **Part 6:** Facilitates analyses that rely on direct access to the full recording such as household co-analysis and circadian rhythm analysis. The reason why it split up in parts is that it avoids having the re-run all preceding parts if you only want to make a small change in the more downstream parts. The specific order and content of the parts has grown for historical with Part 1 and 2 created in 2011-2013, Part 3 and 4 created in 2013-2015, Part 5 created in 2017-2020, and Part 6 created in 2023-2024.
+-   **Part 3:** Estimation of sustained inactivity bouts (rest periods) and sleep, needed as input for Part 4.
+-   **Part 4:** Labels the sustained inactive periods derived in Part 3 as sleep, or sustained inactivity during day time, per night and per file.
+-   **Part 5:** Compiles time series with classification of sleep and physical activity intensity categories by re-using information derived in part 2, 3 and 4. Next, Part 5 generates a descriptive summary metrics such as total time spent in intensity categories, the number of bouts, time spent in bouts, average acceleration per category, and fragmentation.
+-   **Part 6:** Facilitates analyses that rely on direct access to the full recording such as household co-analysis and circadian rhythm analysis. The reason why GGIR is split up in parts is that it avoids having to re-run all preceding parts if you only want to make a small change in the more downstream parts. The specific order and content of the parts have evolved over time, with Parts 1 and 2 developed between 2011 and 2013, Parts 3 and 4 between 2013 and 2015, Part 5 from 2017 to 2020, and Part 6 from 2023 to 2024.
```

Comment:
These revisions are not correct. I intentionally used the word create and not developed, because the development is still ongoing. I want to emphasize the initial creation phase. If that is confusing then maybe refer to these time periods as when the parts were "introduced"?

Reply:
Alright, will use "create" instead.